lib/resty/limit/count.lua

```diff
@@ -41,62 +41,47 @@ function _M.incoming(self, key, commit)
     local limit = self.limit
     local window = self.window
 
-    local remaining, ok, err
+    local count, ok, err
 
     if commit then
-        remaining, err = dict:incr(key, -1, limit)
-        if not remaining then
+        count, err = dict:incr(key, 1, 0, window)
```

Comment:
I think we should support the old version like [core lib](https://github.com/openresty/lua-resty-core/blob/master/lib/resty/core/base.lua#L19), maybe we can have two difference `incoming` for difference version of Openersty